var/spack/repos/builtin/packages/neurodamus-core/package.py

```diff
@@ -3,17 +3,18 @@
 from spack import *
 import os
 import shutil
-
+from spack.fetch_strategy import GitFetchStrategy
 
 class NeurodamusCore(Package):
     """Library of channels developed by Blue Brain Project, EPFL"""
 
     homepage = "ssh://bbpcode.epfl.ch/sim/neurodamus-core"
     git      = "ssh://bbpcode.epfl.ch/sim/neurodamus-core"
 
-    version('develop', git=git, branch='master')
-    version('2.3.3', git=git, tag='2.3.3', preferred=True)
-    version('2.2.1', git=git, tag='2.2.1')
+    version('develop', git=git, branch='master', clean=False)
+    version('2.3.4', git=git, tag='2.3.4', clean=False, preferred=True)
```

Comment:
The newest version is always preferred
```suggestion
    version('2.3.4', git=git, tag='2.3.4', clean=False)
```